app/controllers/calls_controller.rb

```diff
@@ -0,0 +1,77 @@
+class CallsController < ApplicationController
+
+  after_filter :set_header
+
+  # Public: initiates the call process via a request from twillio
+  #
+  # CallSid - default param from twilio (required)
+  def start
+    response = Twilio::TwiML::Response.new do |r|
+      r.Say 'We need you to connect with your congressperson and legislator. We are going to put you in touch with '
+      r.Say 'Whatever other intro text'
+      r.Redirect calls_new_connection_url, method: 'get'
+    end
+
+    render_twiml response
+  end
+
+  # Public: initiates a new outgoing call to congress
+  #
+  # CallSid - default param from twilio (required)
+  def new_connection
+    response = Twilio::TwiML::Response.new do |r|
+      if active_call.target_legislators.any?
+        connection = active_call.create_connection!
+        r.Dial connection.legislator.phone, action: calls_connection_gather_prompt_url
+      else
+        r.Say 'We don\'t have any targets for you. Please try again later.'
+        r.Hangup
+      end
+    end
+
+    render_twiml response
+  end
+
+  # Public: prompts a user reported response on how the connection preformed
+  #
+  # CallSid - active_call's remote_id from twilio(required)
+  # DialCallSid - dialed call's remote_id from twilio (required)
+  def connection_gather_prompt
+    active_connection = active_call.last_connection
+    active_connection.update(remote_id: params['DialCallSid'])
+    response = Twilio::TwiML::Response.new do |r|
+      r.Gather action: calls_connection_gather_url(connection_id: active_connection.id) do |gather|
+        gather.Say 'How did the senator respond? Press 1 for positively, press 2 for negatively'
```

Comment:
Line is too long. [96/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.